cobalt/updater/unzipper.cc

```diff
@@ -0,0 +1,79 @@
+// Copyright 2025 The Cobalt Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "cobalt/updater/unzipper.h"
+
+#include <string>
+#include <utility>
+#include "base/files/file_path.h"
+#include "base/logging.h"
+#include "base/time/time.h"
+#include "third_party/zlib/google/zip.h"
+
+namespace cobalt {
+namespace updater {
+
+namespace {
+
+class UnzipperImpl : public update_client::Unzipper {
+ public:
+  UnzipperImpl() = default;
+
+  void Unzip(const base::FilePath& zip_path,
+             const base::FilePath& output_path,
+             UnzipCompleteCallback callback) override {
+    int64_t time_before_unzip =
+        (base::TimeTicks::Now() - base::TimeTicks()).InMicroseconds();
+    std::move(callback).Run(zip::Unzip(zip_path, output_path));
+    int64_t time_unzip_took_usec =
+        (base::TimeTicks::Now() - base::TimeTicks()).InMicroseconds() -
+        time_before_unzip;
```

Comment:
nit: Can't you just do
```
    auto time_before_unzip = base::TimeTicks::Now();
    std::move(callback).Run(zip::Unzip(zip_path, output_path));
    auto time_unzip_took_usec = 
        (base::TimeTicks::Now() - time_before_unzip).InMilliseconds();
```
? 
(Note the direct conversion to milliseconds, so it's not needed in
l.45-46.
Same in l.53-58 and l.62-63.